packages/aws-cdk-lib/aws-rds/lib/instance.ts

```diff
@@ -713,6 +738,20 @@ export interface DatabaseInstanceNewProps {
    * @default - IPV4
    */
   readonly networkType?: NetworkType;
+
+  /**
+   * The identifier of the CA certificate for this DB instance.
+   *
+   * Specifying or updating this property triggers a reboot.
+   *
+   * For RDS DB engines:
+   * @see https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/UsingWithRDS.SSL-certificate-rotation.html
+   * For Aurora DB engines:
+   * @see https://docs.aws.amazon.com/AmazonRDS/latest/AuroraUserGuide/UsingWithRDS.SSL-certificate-rotation.html
+   *
+   * @default - rds-ca-2019
+   */
+  readonly caCertificateIdentifier?: CertificateIdentifier;
```

Comment:
```suggestion
  readonly caCertificate?: CertificateIdentifier;
```

Can you drop the `Identifier` everywhere please. It's not really relevant to the user intent.
Also in the docstring, e.g. `The the CA certificate used for this DB instance.`